src/whatsnew/3.0.rst

```diff
@@ -0,0 +1,398 @@
+  and replaced by the ``sizes`` object, which contains the improved ``file``,
+  ``active`` and ``external`` size metrics. Fauxton has been updated to match.
+* :ghissue:`2173`: The ability to submit multiple queries against a view using the
+  :method:`POST` to ``/{db}/_design/{ddoc}/_view/{view}`` with the ``?queries=`` option
+  has been replaced by the new ``/db/_design/{ddoc}/_view/{view}/queries`` endpoint.
+  Specify a ``keys`` object when :method:`POST`-ing to this endpoint. [ADD LINK]
+* :ghissue:`2248`: CouchDB externals (``_external/``) have been removed entirely.
+* :ghissue:`2208`: CouchDB no longer supports the ``delayed_commits`` option in the
+  configuration file. All writes are now full commits. The ``/_ensure_full_commit``
+  API endpoint has been retained (as a no-op) for backwards compatibility with old
+  CouchDB replicators.
+* :ghissue:`2395`: The security object in the ``_users`` database cannot be edited by
+  default. A setting exists in the configuration file to revert this behaviour. The
+  ability to override the disable setting is expected to be removed in CouchDB 4.0.
+
+.. _release/3.0.0:
+
+Version 3.0.0
+=============
+
+Features
+--------
+
+.. rst-class:: open
+
+* :ghissue:`1789`: User-defined partitioned databases. Full documentation is here [LINK
+  TBD].
+
+  * These special databases support user-driven placement of documents into the same
+    shard range.
+  * Views and Mango indexes have specific optimizations for partitioned databases as well.
+    [TBD verify Mango specific endpoint or support]
+  * :ghissue:`1842`: Partition size limits. By default, each partition is limited
+    to 10 GiB.
+  * :ghissue:`1684`: Partitioned database support can be disabled via feature
+    flag in ``default.ini``.
+
+* :ghissue:`1972`, :ghissue:`2012`: Automated shard splitting. Databases can now be live
+  re-sharded to increase the ``q`` factor to a larger number. This can be configured
+  to require specific node and range parameters upon execution. Full documentation is
+  here [LINK TBD].
+* :ghissue:`1910`: New automatic view "warming" subsystem, ``ken``. Full documentation
+  is here [LINK TBD].
+* :ghissue:`1904`: Completely new automatic compaction daemon, ``smoosh``. Full
+  documentation is here [LINK TBD].
+* :ghissue:`1889`, :ghissue:`2408`: New IO Queue subsystem. Full documentation is
+  available here [LINK TBD].
+* :ghissue:`2037`: Dreyfus, the CouchDB side of the Lucene-powered search solution,
+  is now shipped with CouchDB. When one or more Clouseau Java nodes are joined to the
+  cluster, text-based indexes can be enabled in CouchDB. It is recommended to
+  have as many Clouseau nodes as you have CouchDB nodes. Search is advertised in the
+  feature list present at ``GET /`` if configured correctly (:ghissue:`2206`). Full
+  documentation is available at [LINK TBD].
+* :ghissue:`2411`: The ``/_up`` endpoint no longer requires authentication when
```

Comment:
/_up can be configured to not require auth even when require_valid_user is enabled.